compiler/test/fail_compilation/issue24246.d

```diff
@@ -0,0 +1,27 @@
+// https://issues.dlang.org/show_bug.cgi?id=24246
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/issue24246.d(116): Error: CTFE internal error: literal `int` could not be copied.
+---
+*/
+
+#line 100
```

Comment:
Why? It just makes it harder to review the test. This `#line` directive should be removed from the language.

Reply:
Habit inflicted by the test runner not letting you write an error inline and then matching the line a la Filecheck.